common/app/views/support/HtmlCleaner.scala

```diff
@@ -426,6 +426,18 @@ case class ImmersiveLinks(isImmersive: Boolean) extends HtmlCleaner {
   }
 }
 
+case class ImmersiveMainEmbed(isImmersive: Boolean, isSixtyDaysModified: Boolean) extends HtmlCleaner {
+  override def clean(document: Document): Document = {
+      if(immersiveMainEmbedSwitch.isSwitchedOn && isSixtyDaysModified && isImmersive) {
+        val srcdoc = document.getElementsByTag("iframe").attr("srcdoc")
+        if(srcdoc != null) {
+            document.getElementsByTag("body").html(srcdoc)
```

Comment:
Does this replace the entire body? I'm not sure that is what we want.

Will this have unexpected consequences on any bit of immersive content created in the last 60 days?